backend/src/services/certificate/certificate-dal.ts

```diff
@@ -75,6 +90,63 @@ export const certificateDALFactory = (db: TDbClient) => {
         query = query.andWhere(`${TableName.Certificate}.commonName`, "like", `%${sanitizedValue}%`);
       }
 
+      if (search) {
+        const sanitizedValue = String(search).replace(new RE2("[%_\\\\]", "g"), "\\$&");
```

Comment:
As we using this every where - why not make it into a reusable function in lib folder